src/main/scala/com/gu/newsletterlistcleanse/services/SttpFactory.scala

```diff
@@ -0,0 +1,24 @@
+package com.gu.newsletterlistcleanse.services
+
+import org.asynchttpclient.DefaultAsyncHttpClientConfig
+import org.asynchttpclient.filter.ThrottleRequestFilter
+import sttp.client.asynchttpclient.WebSocketHandler
+import sttp.client.asynchttpclient.future.AsyncHttpClientFutureBackend
+import sttp.client.{SttpBackend, SttpBackendOptions}
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.concurrent.duration.FiniteDuration
+
+object SttpFactory {
+
+  def createSttpBackend(timeout: FiniteDuration = 5000.seconds, concurrencyLimit: Int = 3): SttpBackend[Future, Nothing, WebSocketHandler] = {
```

Comment:
Same here as above, I think this should be `5.seconds`
```suggestion
  def createSttpBackend(timeout: FiniteDuration = 5.seconds, concurrencyLimit: Int = 3): SttpBackend[Future, Nothing, WebSocketHandler] = {
```